python/graphstorm/gconstruct/transform.py

```diff
@@ -525,6 +551,11 @@ def update_info(self, info):
         self._max_val = max_val
         self._min_val = min_val
 
+        # We need to save the max_val and min_val in the config object.
+        if self._conf is not None:
+            self._conf['max_val'] = self._max_val.tolist()
+            self._conf['min_val'] = self._min_val.tolist()
```

Comment:
was there a bug before? it didn't change the max_val and min_val in conf.

Reply:
Previously, max_val and min_val were not saved back into conf.